dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -1218,4 +1239,118 @@ public void updateUserReferences(String userId, String userRoleId, String replac
 	public void updateStepReferences(String stepId, String replacementStepId) throws DotDataException, DotSecurityException {
 		workFlowFactory.updateStepReferences(stepId, replacementStepId);
 	}
+
+    /**
+     * Filter the list of actions to display according to the user logged permissions
+     * @param actions List of action to filter
+     * @param user User to validate
+     * @param respectFrontEndRoles indicates if should respect frontend roles
+     * @param permissionable ContentType or contentlet to validate special workflow roles
+     * @return List<WorkflowAction>
+     * @throws DotDataException
+     */
+    @CloseDBIfOpened
+	private List<WorkflowAction> filterActionsCollection(final List<WorkflowAction> actions, final User user, final boolean respectFrontEndRoles, final Permissionable permissionable) throws DotDataException {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 205 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)